### ISSUE TITLE
[SOLVED] words added to workspace settings are not taken into account

### ISSUE THREAD

**Issue by opensas** [streetsidesoftware/vscode-spell-checker#2945]
TLDR:

I was working with a [Multi-root workspace](https://code.visualstudio.com/docs/editor/multi-root-workspaces#_settings) and the settings from the `cSpell.words` entry in the `.vscode/settings.json` file of one project was overriding the settings from the `workspace` settings file.

---

I added the `figma` word to my workspace setting:

![image](https://github.com/streetsidesoftware/vscode-spell-checker/assets/481687/eea45db2-a6f1-4a1a-8370-8cf2ff8f195f)

![image](https://github.com/streetsidesoftware/vscode-spell-checker/assets/481687/04bbc849-5964-4303-8935-921382440961)

this is how my `.workspace` file looks like:

```json
{
	"folders": [
		{ "path": "backend" },
		{ "path": "app" },
		{ "path": "billing" },
		{ "path": "docs" }
	],
	"settings": {
		"cSpell.words": [
			"figma"
		]
	}
}
```

but the `figma` word is still marked as Unknown word

Any idea what could be wrong>?

---
vscode Version: 1.84.2
Commit: 1a5daa3a0231a0fbba4f14db7ec463cf99d7768e
Date: 2023-11-09T10:50:47.800Z
Electron: 25.9.2
ElectronBuildId: 24603566
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Linux x64 6.2.0-37-generic

Code Spell Checker v3.0.1

**Comment by LinuxKunaL** [comment]
`.workspace` ? how bro in my vs code the **spell Checker** are created _.vscode_ folder in that contain a `settings.json` look like this : 
![Screenshot from 2023-11-24 12-20-22](https://github.com/streetsidesoftware/vscode-spell-checker/assets/75113218/d1a0ef26-ca8b-4f90-acc6-24d78429e140)

**Comment by Jason3S** [comment]
@opensas,

VSCode allows configuration to be stored in both `.vscode/settings.json` and `*.code-workspace`. See: [multi root workspaces](https://code.visualstudio.com/docs/editor/multi-root-workspaces).

Settings found in `.vscode/settings.json` will replace settings found in `*.code-workspace`. You might be seeing that.

**Comment by opensas** [comment]
thanks I'm having a look at this vscode doc page [Multi-root workspaces settings](https://code.visualstudio.com/docs/editor/multi-root-workspaces#_settings)
just leaving it here for reference
